config/zones/US.yaml

```diff
@@ -0,0 +1,113 @@
+bounding_box:
+  - - -171.791110603
+    - 18.91619
+  - - -66.96466
+    - 71.3577635769
+capacity:
+  battery storage: 4832.0
+  biomass: 14487.1
+  coal: 224217.6
+  gas: 557820.4
+  geothermal: 3838.4
+  hydro: 79372.6
+  hydro storage: 21969.3
+  nuclear: 99886.8
+  oil: 29535.9
+  solar: 64209.1
+  unknown: 1684.2
+  wind: 134695.0
+contributors:
+  - kepiej
+  - KabelWlan
+  - robertahunt
+  - danielmatsuda
+  - brandongalbraith
+  - systemcatch
+  - VIKTORVAV99
+subZoneNames:
+  - US-AK
+  - US-CAL-BANC
+  - US-CAL-CISO
+  - US-CAL-IID
+  - US-CAL-LDWP
+  - US-CAL-TIDC
+  - US-CAR-CPLE
+  - US-CAR-CPLW
+  - US-CAR-DUK
+  - US-CAR-SC
+  - US-CAR-SCEG
+  - US-CAR-YAD
+  - US-CENT-SPA
+  - US-CENT-SWPP
+  - US-FLA-FMPP
+  - US-FLA-FPC
+  - US-FLA-FPL
+  - US-FLA-GVL
+  - US-FLA-HST
+  - US-FLA-JEA
+  - US-FLA-SEC
+  - US-FLA-TAL
+  - US-FLA-TEC
+  - US-HI-HA
+  - US-HI-KA
+  - US-HI-KH
+  - US-HI-LA
+  - US-HI-MA
+  - US-HI-MO
+  - US-HI-NI
+  - US-HI-OA
+  - US-MIDA-PJM
+  - US-MIDW-AECI
+  - US-MIDW-GLHB
+  - US-MIDW-LGEE
+  - US-MIDW-MISO
+  - US-NE-ISNE
+  - US-NW-AVA
+  #- US-NW-AVRN Production only zone, production integrated in US-NW-PACW for the gas and US-NW-BPAT for the wind
+  - US-NW-BPAT
+  - US-NW-CHPD
+  - US-NW-DOPD
+  - US-NW-GCPD
+  - US-NW-GRID
+  # - US-NW-GWA Production only zone, all production integrated with US-NW-NWMT
+  - US-NW-IPCO
+  - US-NW-NEVP
+  - US-NW-NWMT
+  - US-NW-PACE
+  - US-NW-PACW
+  - US-NW-PGE
+  - US-NW-PSCO
+  - US-NW-PSEI
+  - US-NW-SCL
+  - US-NW-TPWR
+  - US-NW-WACM
+  - US-NW-WAUW
+  #- US-NW-WWA Production only zone, all production integrated with US-NW-NWMT
+  - US-NY-NYIS
+  - US-SE-AEC
+  - US-SE-SEPA
+  - US-SE-SOCO
+  - US-SW-AZPS
+  #- US-SW-DEAA Production only zone, all production integrated with US-SW-SRP
+  - US-SW-EPE
+  #- US-SW-GRIF Production only zone, all production integrated with US-SW-WALC
+  #- US-SW-HGMA Production only zone, all production integrated with US-SW-SRP
+  - US-SW-PNM
+  - US-SW-SRP
+  - US-SW-TEPC
+  - US-SW-WALC
+  - US-TEN-TVA
+  - US-TEX-ERCO
+bypassedSubZones:
```

Comment:
I assume this is for zones with no parsers?
Just for my own future reference.

Reply:
Exactly, or zones that have been depreciated so no data comes in from the parser but we don't know yet with which we should merge it. (US-MIDW-GLHB) in this case.